src/commons/dropdown/LocaleSelector.tsx

```diff
@@ -0,0 +1,41 @@
+import { FormGroup, HTMLSelect, Icon, PopoverPosition, Tooltip } from '@blueprintjs/core';
+import { IconNames } from '@blueprintjs/icons';
+import { useState } from 'react';
+import i18n from 'src/i18n/i18n';
+import { i18nLanguageCode, resources } from 'src/i18n/locales';
+
+const languageOptions = Object.keys(resources)
+  .filter(String)
```

Comment:
Can I check what's this filter guarding against?

Reply:
because we conditionally spread the psuedo language resources into `resources`, there could be an undefined` key if we spread an empty object `{ }`

im not _entirely_ certain of this behavior though, the type hints suggests that this could be a possibility, hence the filter.